contracts/Cashier.sol

```diff
@@ -105,7 +105,6 @@ contract Cashier is ICashier, ReentrancyGuard, Ownable, Pausable {
         voucherKernel = _voucherKernel;
         voucherSetTokenAddress = _voucherSetTokenAddress;
         voucherTokenAddress = _voucherTokenAddress;
-        disasterState = false;
     }
```

Comment:
This constructor is still missing events. Same for the VoucherKernel and Cashier constructors

Reply:
Thanks for mentioning this. I have now added missing events to the constructor in Cashier.sol, BosonRouter.sol and VoucherKernel.sol with unit tests.